release-notes/opensearch-release-notes-2.13.0.md

```diff
@@ -0,0 +1,768 @@
+<h1>OpenSearch and OpenSearch Dashboards 2.13.0 Release Notes</h1>
+<h2>RELEASE HIGHLIGHTS</h2>
+
+<h2>NEW FEATURES</h2>
+
+<h2>EXPERIMENTAL FEATURES</h2>
+
+<h2>RELEASE DETAILS</h2>
+<ul>
+<li>[OpenSearch and OpenSearch Dashboards 2.13.0](https://opensearch.org/versions/opensearch-2-3-0.html) includes the following features, enhancements, bug fixes, infrastructure, documentation, maintenance and refactoring updates.
+</li>
+<li>
+OpenSearch <a href="https://github.com/opensearch-project/OpenSearch/blob/2.12/release-notes/opensearch.release-notes-2.13.0.md">Release Notes</a>.
```

Comment:
there is typo in link 2.12 to be replaced by 2.13

https://github.com/opensearch-project/OpenSearch/blob/2.13/release-notes/opensearch.release-notes-2.13.0.md

Reply:
Thanks! Fixed it.